### DISCUSSION TITLE
Broken: TN news ( Todo Noticias )  Argentina

### DISCUSSION THREAD

**Discussion by eze92** [iptv-org/community#593] [Q&A]
This would be the YouTube channel, if you can take it from there https://www.youtube.com/live/cb12KmMMDJA?si=8kIrOf6Y2i4bPEgu

**Comment by BellezaEmporium** [comment]
https://ythls.armelin.one/video/cb12KmMMDJA.m3u8 ?

**Comment by eze92** [comment]
> https://ythls.armelin.one/video/cb12KmMMDJA.m3u8 ?

Yeah ! is that iptv

**Comment by eze92** [comment]
The channel is still broken, I'm from Argentina until a month ago it worked
